notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>nit: can you reduce the size of this image? I don't think it has to be so big for an example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Makes sense! Thank you.</p>